stream-chat-android/src/main/java/com/getstream/sdk/chat/adapter/ChannelListDiffCallback.java

```diff
@@ -67,19 +65,12 @@ public boolean areContentsTheSame(int oldItemPosition, int newItemPosition) {
     @Nullable
     @Override
     public Object getChangePayload(int oldItemPosition, int newItemPosition) {
-
         ChannelItemPayloadDiff diff = new ChannelItemPayloadDiff();
 
         Channel oldChannel = oldList.get(oldItemPosition);
         Channel newChannel = newList.get(newItemPosition);
 
-        Message oldLastMessage = computeLastMessage(oldChannel);
-        Message newLastMessage = computeLastMessage(newChannel);
-
-        if (oldLastMessage != null && newLastMessage != null) {
-            diff.lastMessage = !oldLastMessage.getId().equals(newLastMessage.getId());
-        }
-
+        diff.lastMessage = !lastMessagesAreTheSame(oldChannel, newChannel);
```

Comment:
While we're here, maybe it's time to move the `lastMessagesAreTheSame` to this callback class from `LlcMigrationUtils`? Only being used here as far as I see.